From 1db5fad181298a406ea038154199e3f14b968ea4 Mon Sep 17 00:00:00 2001
From: abhishek262.kumar <Abhishek262.Kumar@ril.com>
Date: Thu, 27 Apr 2023 13:32:12 +0530
Subject: [PATCH] [AZ-216603] Implementation of Factory reset using hard reset key [Part 2]

Do factory reset directly if HW reset key is pressed for >= 9 secs

Change-Id: I8419cab2a414d23ff3d2dfe8f620e3a0352f71ee
---

diff --git a/arch/arm64/boot/dts/amlogic/sc2_s905x4_ah212_drm.dts b/arch/arm64/boot/dts/amlogic/sc2_s905x4_ah212_drm.dts
index f9f0dda..b8d6608 100644
--- a/arch/arm64/boot/dts/amlogic/sc2_s905x4_ah212_drm.dts
+++ b/arch/arm64/boot/dts/amlogic/sc2_s905x4_ah212_drm.dts
@@ -401,7 +401,7 @@
 		scan_period = <20>;
 		key_num = <1>;
 		key_name = "power";
-		key_code = <116>;
+		key_code = <249>;
                 key_type = <EV_KEY EV_SW>;
 		key-gpios = <&gpio  GPIOE_2  GPIO_ACTIVE_HIGH>;
 		detect_mode = <0>;/*0:polling mode, 1:irq mode*/
